lib/saluki-config/Cargo.toml

```diff
@@ -6,8 +6,12 @@ license = { workspace = true }
 repository = { workspace = true }
 
 [dependencies]
+arc-swap = { workspace = true }
 figment = { workspace = true, features = ["env", "json", "yaml"] }
+http = { workspace = true }
+reqwest = { workspace = true, features = ["rustls-tls", "json"] }
```

Comment:
Can you change `rustls-tls` to `rustls-tls-native-roots-no-provider`?

This will avoid pulling in some extra dependencies that we don't need.